credentialMixin.js

```diff
@@ -40,7 +40,11 @@ export const credentialMixin = {
     clickable: {
       type: Boolean,
       required: false
-    }
+    },
+    profile: {
+      type: Object,
+      required: false,
```

Comment:
This seems to conflict with previous code that depends on `profile.name`. We are expecting `profile.name` to exist. We must make profile required here and then validate that `profile.name` exists.